modules/videoMediator/videoMediator.py

```diff
@@ -0,0 +1,20 @@
+import TargetAcquisition
+import DecklinkSRC
+
+class VideoMediator:
+    tent_coordinates = None
```

Comment:
Minor thing before merge, we standardize on camelCase for all variable names. Therefore target_acquisition should be targetAcquisition.